### PR TITLE
Display distance to car

### PR DESCRIPTION
**Pull request by yunikkk**

### CODE REVIEW

File: MapboxVision/src/main/java/com/mapbox/vision/visionevents/events/roaddescription/RoadDescription.kt [hunk 1]
```diff
@@ -25,70 +24,85 @@ data class RoadDescription(
...
+                                rightMarking = Marking(
+                                        rightMarkingType,
+                                        worldPoints = emptyList()// TODO fill lane points
+                                )
+                        )
+                )
             }
 
-            return RoadDescription(roadDescriptionDataBuffer.roadDescriptionIdentifier, lines, visibleLeftLanes, egoOffset)
+            return RoadDescription(
```
Comment (BenDenen) [inline MapboxVision/src/main/java/com/mapbox/vision/visionevents/events/roaddescription/RoadDescription.kt]: Why did you change it. Do you need roadDescription object?

Comment (yunikkk) [reply MapboxVision/src/main/java/com/mapbox/vision/visionevents/events/roaddescription/RoadDescription.kt]: Yeah I needed width in previous distance to car version - but now use updated world description so dont need roaddescription. However - I've aligned it with iOS implementation already.
